lib/elixir/test/elixir/kernel/expansion_test.exs

```diff
@@ -1081,7 +1081,8 @@ defmodule Kernel.ExpansionTest do
           fn -> 17 end
         end
 
-      assert expand(before_expansion) == after_expansion
```

Comment:
without that change this test fails because after expansion there is `:no_parens`. Is it OK?
```
{:__block__, [], [Kernel.ExpansionTarget, {:fn, [{:no_parens, true}], [{:->, [{:no_parens, true}], [[], 17]}]}]}
```

Reply:
Yes, this is better.